Gulpfile.js

```diff
@@ -9,30 +9,66 @@ ugifly = require('gulp-uglify');
 const libs = 'app/public/libs/'; 
 
 gulp.task('swigit', gulpSequence('lint', 'concat', ['minify-css', 'uglify']); 
+const gulp = require('gulp'),
+gulpSequence = require('gulp-sequence'),
+gulpIgnore = require('gulp-ignore'),
+jshint = require('gulp-jshint'),
+cleanCSS = require('gulp-clean-css')
+concat = require('gulp-concat'),
+uglify = require('gulp-uglify'); 
+
+const libs = './app/public/libs/'; 
+
+gulp.task('default', gulpSequence('concat', 
+  'lint', 
+  // 'minify-css'
+  ['minify-css', 'uglify']
+  )); 
+
+//For some reason, linting doesn't work if I pass in all our JS files.
+
+//I have it lint the concatenated file instead, which doesn't seem to hurt
+//the speed too much. 
 
 gulp.task('lint', function(){
-  return gulp.src(['./**/*.js']) //Takes in everything that's *not* in our public/lib folder 
+  return gulp.src(
+  'app/public/build/js') //Takes in everything that's *not* in our public/lib folder 
```

Comment:
Since app/public/build is only going to contain our concatinated, uglified js files, and will contain both our dependencies & application logic, we probably don't want to be linting anything in there.

We should be linting everything inside of the /public & /server directories, excluding /build & /libs, so our src array will probably look something like this:

`['./app/server/**/*.js','./app/public/**/*.js','!./app/public/libs','!./app/public/build]`